source/dotnet/Library/AdaptiveCards/Rendering/AdaptiveHostConfig.cs

```diff
@@ -156,5 +161,56 @@ public string ToJson()
         {
             return JsonConvert.SerializeObject(this, Formatting.Indented);
         }
+
+        public string GetFontFamily(AdaptiveFontStyle fontStyle)
```

Comment:
could use some documentation on new functions

Reply:
Added some comments. Not sure how specific we usually are so let me know what you think.